packages/gguf/src/gguf.ts

```diff
@@ -384,14 +384,17 @@ export async function gguf(
 		});
 	}
 
+  	const alignment		=metadata['general.alignment']??32;
+	const tensorsOffset	=offset+(alignment-(offset%alignment));
```

Comment:
This calculation is not correct because the current `offset` is the offset of the current tensor metadata, not the offset of whole metadata

Reply:
current offset is taken after tensor metadata ends, that's where starts actual tensors. also, i did check the value received with the same value reported by gguf tool from llama.cpp, it matched. 
https://github.com/ggerganov/ggml/blob/master/docs/gguf.md

so i don't really understand why you believe it's wrong :).

Reply:
ok sorry I misread the code. I thought that here you're setting it for each tensor. It's correct to use the `offset` only after we're done reading all data.